components/org.wso2.carbon.identity.auth.abstractotp/src/main/java/org/wso2/carbon/identity/auth/abstractotp/model/OTP.java

```diff
@@ -0,0 +1,46 @@
+package org.wso2.carbon.identity.auth.abstractotp.model;
```

Comment:
license. Check all the files in the project.

Reply:
Added the license header to all files.